docs/api/saved-objects/delete.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[saved-objects-api-delete]]
+=== Delete Object
+
+experimental[This functionality is *experimental* and may be changed or removed completely in a future release.]
+
+The delete saved object API permanently removes a Kibana saved object. Once a
+saved object has been deleted, _it cannot be recovered_.
+
+==== Request
+
+`DELETE /api/saved_objects/<type>/<id>`
+
+==== Path Parameters
+
+`type` (required)::
+  (string) Valid options, include: `visualization`, `dashboard`, `search`, `config`, and `timelion-sheet`
+
+`id` (required)::
+  (string) Object ID being removed
+
+
+==== Examples
+
+The following example deletes an index pattern object with an ID of `my-pattern`
+
```

Comment:
`index-pattern` is used in the example but isn't one of the valid options above.